hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestSCMNodeManager.java

```diff
@@ -17,53 +17,40 @@
  */
 package org.apache.hadoop.hdds.scm.node;
 
-import java.io.File;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Set;
-import java.util.UUID;
-import java.util.concurrent.ScheduledFuture;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
-
```

Comment:
Please don't move around imports unnecessarily.  It inflates the patch and makes harder to review.  It also increases probability of conflict for backports.

Reply:
Thank you for your comment, I have updated the code, please review it again